app/models/email_thread.rb

```diff
@@ -6,5 +6,5 @@ class EmailThread < ApplicationRecord
   has_many :messages
   has_many :actions, through: :messages
 
-  enum urgency: { none: 0, low: 1, medium: 2, high: 3, urgent: 4 }
+  enum urgency: { none: 0, low: 1, medium: 2, high: 3, urgent: 4 }, _suffix: true
```

Comment:
Had to add suffix as without it it was throwing error

ArgumentError: You tried to define an enum named "urgency" on the model "EmailThread", but this will generate a class method "none", which is already defined by Active Record. (ArgumentError)

Reply:
Good solution! I was honestly just going to rename "none" but this is fine haha